github/issues_assignees.go

```diff
@@ -44,3 +44,45 @@ func (s *IssuesService) IsAssignee(owner string, repo string, user string) (bool
 	assignee, err := parseBoolResponse(err)
 	return assignee, resp, err
 }
+
+// AddAssignees adds the provided GitHub users as assignees to the issue.
+//
+// GitHub API docs: https://developer.github.com/v3/issues/assignees/#add-assignees-to-an-issue
+func (s *IssuesService) AddAssignees(owner, repo string, number int, assignees []string) (*Issue, *Response, error) {
```

Comment:
I wanted to check if we already have a precedent on API style for endpoints that take a string slice as a last parameter, whether to do `, assignees []string)` here or `, assignees ...string)`.

I've confirmed we do, and this is consistent with the style used there.

https://godoc.org/github.com/google/go-github/github#UsersService.AddEmails
https://godoc.org/github.com/google/go-github/github#UsersService.DeleteEmails
https://godoc.org/github.com/google/go-github/github#IssuesService.AddLabelsToIssue
https://godoc.org/github.com/google/go-github/github#IssuesService.ReplaceLabelsForIssue